webapp/src/main/webapp/components/snackbar/ExoNewsNotificationAlert.vue

```diff
@@ -51,9 +51,12 @@ export default {
   },
   watch: {
     displayAlert() {
+      const deletedNews = localStorage.getItem('deletedNews');
       if (!this.displayAlert) {
         this.$emit('dismissed');
-        this.$root.$emit('news-deleted');
+        if (deletedNews != null) {
```

Comment:
This component should still a common one and independent from the type of alert (delete, share ...)